.nuspec/Microsoft.Maui.Controls.SingleProject.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup Condition=" '$(SingleProject)' == 'true' and '$([MSBuild]::GetTargetPlatformIdentifier($(TargetFramework)))' == 'windows' ">
     <OutputType Condition="'$(OutputType)' == 'Exe'">WinExe</OutputType>
     <WindowsPackageType Condition=" '$(WindowsPackageType)' == '' and '$(EnableMsixTooling)' == 'true' and ('$(OutputType)' == 'Exe' or '$(OutputType)' == 'WinExe') ">MSIX</WindowsPackageType>
+    <WindowsAppSDKSelfContained Condition=" '$(WindowsAppSDKSelfContained)' == '' and '$(WindowsPackageType)' == 'None' and ('$(OutputType)' == 'Exe' or '$(OutputType)' == 'WinExe') ">true</WindowsAppSDKSelfContained>
```

Comment:
Can we encapsulate `('$(OutputType)' == 'Exe' or '$(OutputType)' == 'WinExe')` so the condition isn't duplicated on all these lines?

Also, can OutputType ever be `Exe`? On line 4 we say if OutputType is `Exe` we are changing it to `WinExe`.